static_report/src/utils/amplitudeEvents.ts

```diff
@@ -19,7 +19,11 @@ export function amplitudeTrack({
   const API_KEY = window.PIPERIDER_METADATA.amplitude_api_key;
   const PROJECT_ID = window.PIPERIDER_METADATA.amplitude_project_id;
 
-  if (!API_KEY && process.env.NODE_ENV === 'development') {
+  if (
+    !API_KEY ||
+    process.env.NODE_ENV === 'development' ||
+    process.env.REACT_APP_E2E === 'true'
```

Comment:
We're building our application by prod build, so `NODE_ENV` cannot override manually, so I added a `REACT_APP_E2E` env for recognition.
 
> when you run npm run build to make a production bundle, it is always equal to 'production'. **You cannot override `NODE_ENV` manually.**  ([reference](https://create-react-app.dev/docs/adding-custom-environment-variables/))